lib/routes/censorbib/namespace.ts

```diff
@@ -0,0 +1,6 @@
+import type { Namespace } from '@/types';
```

Comment:
Please move the [namespace](https://docs.rsshub.app/joinus/new-rss/start-code#creating-namespace) to `nymity` instead.